TrueDialog.Client.Tests/App.config

```diff
@@ -1,31 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <configSections>
-    <section name="trueDialog" type="TrueDialog.Configuration.TrueDialogConfigSection, TrueDialog.Client"/>
+    <section name="trueDialog" type="TrueDialog.Configuration.TrueDialogConfigSection, TrueDialog.Client.Standard"/>
   </configSections>
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
     </startup>
   <trueDialog baseUrl="https://api.truedialog.com/api/v2.1/">
     <!-- TODO: Fill in with your login information. -->
     <auth username="" password="" defaultAccountId="" apiKey="" apiSecret=""/>
-    <!-- Try up to 3 times once every 1 second-->
-    <retryPolicy type="TrueDialog.Retry.Strategy.FixedRetryStrategy" maxTries="3" interval="0:0:1"/>
-
-    <!-- Try up to 4 times retring after: .5, 1, 1.5 seconds. -->
-    <!--
-        <retryPolicy
-            type="ThreeSeventy.Vector.Client.IncrementalRetryStrategy"
-            maxTries="4"
-            interval="0:0:0.5" />
-        -->
-
-    <!-- Try up to 5 times retring after: .5, 1, 2, 4 seconds -->
-    <!--
-        <retryPolicy
-            type="ThreeSeventy.Vector.Client.ExponentialRetryStrategy"
-            maxTries="5"
-            interval="0:0:0.5" />
-        -->
```

Comment:
This was here for documentation purposes.  Any particular reason why it's been removed?